### PR TITLE
qtemu: Add version 2.1

### LINKED ISSUE #7769: [Request] QtEmu
### Information

**Name:** QtEmu

**Description:** Graphical User Interface for QEMU written in Qt

**Homepage:** https://gitlab.com/qtemu/gui

**Download link(s):** https://www.carlavilla.es/qtemu/qtemu_portable_x86_64.zip

**Some indication of popularity/repute:** 65 Stars on GitLab

### PR DESCRIPTION
**Pull request by LazyGeniusMan**
Closes #7769 

Relates to #XXXX

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/qtemu.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "description": "Graphical User Interface for QEMU written in Qt",
+    "homepage": "https://gitlab.com/qtemu/gui",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.carlavilla.es/qtemu/qtemu_portable_x86_64.zip",
+            "hash": "7999d54cc3b433de5299a5644e4d3f9d073636b243d2a3335af478c7fadd2e0e"
+        }
+    },
+    "extract_dir": "qtemu_portable_x86_64",
```
Comment (rashil2000) [inline bucket/qtemu.json:12]: Move this to arch 64bit block

**Review by rashil2000 — CHANGES_REQUESTED** [review]
Anything to persist?

**Comment by LazyGeniusMan** [comment]
> Anything to persist?

Nope, I seen nothing added after I configure the app and add a VM, probably it's on %APPDATA% for setting and %USER% for VM

**Pull request merged**